src/transformState.js

```diff
@@ -6,6 +6,26 @@
  */
 function transformState(state, actions) {
   // write code here
+
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
+      Object.assign(state, action.extraData);
+    }
```

Comment:
Checklist 2, Use `switch` statement if you have a limited amount of conditions.

Reply:
allright. forgot about it